cea/datamanagement/terrain_helper.py

```diff
@@ -80,6 +80,14 @@ def terrain_elevation_extractor(locator, config):
     x_max = bounding_box_district_file[2] + extra_border
     y_max = bounding_box_district_file[3] + extra_border
 
+    # make sure output is a whole number when min-max is divided by grid size
```

Comment:
i'm curious... can't you just use `math.ceil(x_max)` and `math.floor(x_min)` to get to the next integer?

Reply:
Oh, what I meant was that the difference of `max` and `min` i.e. `max - min` would be a multiple of `grid_size`, so that the result would be a whole number when divided by `grid_size`.

Reply:
it's because of these lines, which truncates the decimals when cast to `int`. Some times it causes the terrain boundary to have one less row/column then it's supposed to.

https://github.com/architecture-building-systems/CityEnergyAnalyst/blob/e59f12da4015c7cb31b8be47dcbb5ed45d6cc03b/cea/datamanagement/terrain_helper.py#L102-L103

Cound just do `math.ceil()`, but the terrain boundary would be lopsided i.e. right boundary more than the left, so instead I added the difference on both ends.

Reply:
thank you for explaining!